src/visit_vtk/full/CMakeLists.txt

```diff
@@ -129,6 +49,75 @@ else()
         VTK::ImagingHybrid VTK::IOImage VTK::tiff)
 endif()
 
-target_link_libraries(visit_vtk lightweight_visit_vtk ${vtklibs} ${OPENEXR_LIB})
 
-VISIT_INSTALL_TARGETS(visit_vtk)
+blt_add_library(
+    NAME        visit_vtk
+    SOURCES     InitVTK.C
+                vtkAxisDepthSort.C
```

Comment:
Ok, so these source file name listings are also not prefixed by `${CMAKE_CURRENT_SOURCE_DIR}/`. Not sure what rationale is except maybe because we don't need releative paths here?

Reply:
When you call `blt_add_library` in the same directory as source, the full-path-to-source isn't needed.
When source files live in subdirectories, and they are appended to a list as I have done, then full path is needed, hence use of ${CMAKE_CURRENT_SOURCE_DIR} in those cases.

A case could be made for consistency, and use of CMAKE_CURRENT_SOURCE_DIR everywhere.